src/main/java/thrift/logic/LogicManager.java

```diff
@@ -107,6 +108,8 @@ public void parseRefreshable(Command command, BalanceBar balanceBar) {
             model.updateExpenseForCurrentMonth();
             model.updateIncomeForCurrentMonth();
             updateBalanceBar(balanceBar);
+        } else {
```

Comment:
Why do you use an `else` just for an `assert`? It is not wrong, just seems strange.

Reply:
This is an attempt to improve code quality by [following the textbook](https://nus-cs2103-ay1920s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-the-default-branch).

I have added a logging statement to improve clarity.

Reply:
The assertion has now gained meaning, nice!